manifest.yml

```diff
@@ -2,16 +2,16 @@
 declared-services:
   dashDB-nodesample:
     label: dashDB
```

Comment:
I am hoping the label would need to be changed to "dashDB For Transactions"

Reply:
good point! updated

Reply:
Thank you.